From 8245bb925238ba8f7ffd1abd8737471653cecaf5 Mon Sep 17 00:00:00 2001
From: Matej Habrnal <mhabrnal@redhat.com>
Date: Mon, 6 Aug 2018 13:32:01 +0200
Subject: [PATCH] potfiles: fix issue in POTFILES.in

src/plugins/report_Bugzilla.xml.in doesn't exist anymore,
it was rename to src/plugins/report_Bugzilla.xml.in.in.
Actualize POTFILES files regarding to this change.

Related to #1449646

Signed-off-by: Matej Habrnal <mhabrnal@redhat.com>
---
 po/POTFILES.in   | 2 +-
 po/POTFILES.skip | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/po/POTFILES.in b/po/POTFILES.in
index 9f972aef..fcc02184 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -29,7 +29,7 @@ src/lib/problem_report.c
 src/lib/reported_to.c
 src/lib/run_event.c
 src/plugins/abrt_rh_support.c
-src/plugins/report_Bugzilla.xml.in
+src/plugins/report_Bugzilla.xml.in.in
 src/plugins/report.c
 src/plugins/reporter-bugzilla.c
 src/plugins/reporter-kerneloops.c
diff --git a/po/POTFILES.skip b/po/POTFILES.skip
index 5268b71d..cd1a48c3 100644
--- a/po/POTFILES.skip
+++ b/po/POTFILES.skip
@@ -1,3 +1,3 @@
 contrib/command-not-found/pk-tools-common.c
 build/
-src/plugins/report_Bugzilla.xml.in.in
+src/plugins/report_Bugzilla.xml.in
-- 
2.18.0

